db/fixtures/miq_product_features.yml

```diff
@@ -2440,34 +2440,34 @@
   - :name: Dialogs
     :description: Dialogs Accordion
     :feature_type: node
-    :hidden: true
+    :hidden: false
     :identifier: dialog_accord
     :children:
     - :name: Modify
       :description: Modify Dialogs
       :feature_type: admin
       :identifier: dialog_admin
-      :hidden: true
+      :hidden: false
       :children:
       - :name: Add
         :description: Add Dialog in the Dialog Editor
-        :feature_type: admin
-        :hidden: true
+        :feature_type: tenant_node
```

Comment:
Perhaps it would be better to just add a new column for this instead of overloading `feature_type`. Maybe a boolean column called `per_tenant`? I'll leave it up to you for the implementation.